services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int,) -> None:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=movie,
+        cinema_hall=cinema_hall
+
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> None:
```

Comment:
The function `get_movies_sessions` is annotated to return `None`, but it actually returns a `QuerySet` of `MovieSession` objects. Consider changing the return type annotation to `QuerySet[MovieSession]`.